app/js/components/manageApps/ManageApps.jsx

```diff
@@ -630,12 +652,12 @@ export default class ManageApps extends React.Component {
               <h2 className="manage-addon-title">Add-on Manager</h2>
               <span className="pull-right manage-settings-wrapper">
                 <span id="startall-modules-btn"
-                  className="btn btn-secondary" 
```

Comment:
I recommend that you only make changes that directly affect the ticket directly, ie. formatting changes should be done separately. They make the PR a little noisy

Reply:
will take care of that next time. thanks